dev-packages/application-package/src/extension-package.ts

```diff
@@ -24,6 +24,7 @@ export interface Extension {
     frontendElectron?: string;
     backend?: string;
     backendElectron?: string;
+    backendMasterElectron?: string;
```

Comment:
What are expectations about js fie, the same as for other entry points? Does it return `ContainerModule` and use inversify?

Reply:
As it stands it doesn't use `ContainerModule` or inversify.
Reason being I would like to see how the use cases for this entry point evolve and didn't want to impose any restrictions. For now each `backendMasterElectron` package can use it's own inversify containers but a simple js entry point would do for `electron-updates` (or any modules required to run from a different thread) to work.

Reply:
It's decided to Theia to use inversify in all entry points to allow customizations. I don't think this entry point should deviate.

Reply:
Can we use this entry point as a short term solution and replace it with an inversify method later?